cpp/ql/lib/semmle/code/cpp/ir/dataflow/internal/DataFlowPrivate.qll

```diff
@@ -6,6 +6,7 @@ private import semmle.code.cpp.ir.internal.IRCppLanguage
 private import SsaInternals as Ssa
 private import DataFlowImplCommon as DataFlowImplCommon
 private import codeql.util.Unit
+import Node0ToString
```

Comment:
Is there a reason thi sis not a private import?

Reply:
Slightly, yes. It's needed [here](https://github.com/github/codeql/pull/15088/files#diff-e9e4c8dbaa54c9f16e1533c09bb66ff3ec456e747f98de139672cfa0412a463cR489). Alternatively, we can private import `Node0ToString` here _and_ in `DataFlowUtil`. But since `DataFlowPrivate` isn't exported out of the dataflow library (unlike `DataFlowUtil`) I think it's fine. If you disagree I'm happy to change it, though

Reply:
Alright. I'll do that!

Reply:
Fixed in fcc3113bfc7b0a543e670f2866345d2356a2cf1a. I also noticed that `isDebugMode` computed `isNotDebugMode`. That has now been fixed in 401ab3b0358708838912824dcb1316baad02f038

Reply:
Thanks.